### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Language, link, and consistency updates for the CLI README.

### CODE REVIEW

**Review by jason-seqera — APPROVED** [review]
This is fantastic, thanks!

File: README.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
-# Nextflow Tower CLI
+# tw CLI
 
-`tw` is [Tower](https://tower.nf/) on the command line. It brings Tower concepts including Pipelines, Actions and Compute Environments to the terminal.
+`tw` is [Seqeta Platform](https://cloud.seqera.io/) on the command line. It brings Seqera concepts like pipelines, actions, and compute environments to the terminal.
```
Comment (mbosio85) [inline README.md:3]: Typo in Seqet*a

**Review by mbosio85 — CHANGES_REQUESTED** [review]
Just a minor typo at the beginning, the rest is ok for me!

File: README.md (hunk 1 above)
Comment (llewellyn-sl) [reply README.md:3]: Oof thanks for catching that.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Seqera CLI
+# tw CLI
 
 `tw` is [Seqeta Platform](https://cloud.seqera.io/) on the command line. It brings Seqera concepts like pipelines, actions, and compute environments to the terminal.
```
Comment (mbosio85) [inline README.md]: Minor typo it's tin the first link it says Seqe**t**a instead of Seqera

**Review by mbosio85 — APPROVED** [review]

**Pull request merged**